### PR TITLE
Move hard-coded constants to global consts

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
Best practices

**What does this PR do? Why is it needed?**
This PR is part of #6337, changes most hard-coded constants to a global const at the top of the relevant files.

### CODE REVIEW

File: validator/main.go [hunk 1]
```diff
@@ -34,6 +34,8 @@ import (
 	"google.golang.org/grpc"
 )
 
+const connTimeout = 10 * time.Second /* Cancel if cannot connect to beacon node in 10 seconds. */
```
Comment (farazdagi) [inline validator/main.go:37]:
```suggestion
// connTimeout defines a period after which connection to beacon node is cancelled.
const connTimeout = 10 * time.Second
```

Normally, we are using this type of commenting everywhere.

File: validator/accounts/v1/status.go [hunk 2]
```diff
@@ -13,6 +13,8 @@ import (
 	"go.opencensus.io/trace"
 )
 
+const statusTimeout = 30 * time.Second /* Cancel if running over thirty seconds. */
```
Comment (farazdagi) [inline validator/accounts/v1/status.go:16]:
```suggestion
// statusTimeout defines a period after which request to fetch account statuses is cancelled.
const statusTimeout = 30 * time.Second
```

Consider updating comment.

File: shared/slotutil/slottime.go [hunk 3]
```diff
@@ -35,5 +37,5 @@ func EpochsSinceGenesis(genesis time.Time) uint64 {
 // in milliseconds, useful for dividing values such as 1 second into
 // millisecond-based durations.
 func DivideSlotBy(timesPerSlot int64) time.Duration {
-	return time.Duration(int64(params.BeaconConfig().SecondsPerSlot*1000)/timesPerSlot) * time.Millisecond
+	return time.Duration(int64(params.BeaconConfig().SecondsPerSlot*msPerSecond)/timesPerSlot) * time.Millisecond
```
Comment (farazdagi) [inline shared/slotutil/slottime.go:40]: Probably better to use `time.Millisecond`?

File: shared/event/subscription.go [hunk 4]
```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/mclockutil"
 )
 
+const waitQuotient = 10
```
Comment (farazdagi) [inline shared/event/subscription.go:27]: We need some comment explaining what this is for.

File: endtoend/components/validator.go [hunk 5]
```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/testutil"
 )
 
+const depositGasLimit = 500000
```
Comment (farazdagi) [inline endtoend/components/validator.go:28]: Is it ok that original amount was 4000000?

File: beacon-chain/p2p/service.go [hunk 6]
```diff
@@ -54,6 +54,11 @@ const lookupLimit = 15
 // maxBadResponses is the maximum number of bad responses from a peer before we stop talking to it.
 const maxBadResponses = 5
 
+// Exclusion list cache config values.
+const cacheNumCounters = 1000
+const cacheMaxCost = 1000
+const cacheBufferItems = 64
+
```
Comment (farazdagi) [inline beacon-chain/p2p/service.go:61]: Please consider combining into a single `const` (if they have a single comment as a group - they can, probably, be combined)

File: endtoend/components/validator.go (hunk 5 above)
Comment (0xKiwi) [reply endtoend/components/validator.go:28]: The deposit contract has a ceiling of gas usage of around 500K, maybe I should leave it as is though :+1:

**Pull request merged**